cranelift/codegen/src/isa/aarch64/abi.rs

```diff
@@ -151,6 +151,33 @@ impl ABIMachineSpec for AArch64MachineDeps {
         let max_per_class_reg_vals = 8; // x0-x7 and v0-v7
         let mut remaining_reg_vals = 16;
 
+        let ret_area_ptr = if add_ret_area_ptr {
+            debug_assert!(args_or_rets == ArgsOrRets::Args);
```

Comment:
```suggestion
            debug_assert_eq!(args_or_rets, ArgsOrRets::Args);
```

for slightly better panic messages

Reply:
There are only two options in this enum and this matches the other debug assertions, but sure.